### DISCUSSION TITLE
How to add an icon to tabs without an index.md?

### DISCUSSION THREAD

**Discussion by couleurm** [squidfunk/mkdocs-material#6419] [Q&A] [answered]
Hey, is there a way for a navigation tab to have an icon without needing to make an index.md with an icon in it?

**Comment by squidfunk** [comment]
No. You can't add icons to sections, because there's no associated Markdown file. It might be implementable with a plugin or a hook, but I don't think somebody has done this yet.

**Comment by kamilkrzyskow** [comment, marked as answer]
Not by config change, or some easy straightforward change, but I think you should be able to get the result you want via theme overrides.

Either in the `tabs.html` or `tabs-item.html` you could add a conditional to detect the name / path of the rendered item and add an icon yourself that way.

https://squidfunk.github.io/mkdocs-material/customization/#extending-the-theme
